### ISSUE TITLE
Build fail on Arch Linux

### ISSUE THREAD

**Issue by onoketa** [cloudflare/tls-tris#175]
```bash
[onoketa@arch-desktop tls-tris]$ make -f _dev/Makefile build-all
rm -rf /home/onoketa/tls-tris/_dev/GOROOT/linux_amd64
mkdir -p "/home/onoketa/tls-tris/_dev/GOROOT/linux_amd64/pkg"
# macOS cp requires -R instead of -r. It makes no difference for GNU cp.
cp -HR /usr/lib/go/src /home/onoketa/tls-tris/_dev/GOROOT/linux_amd64/src
cp -HR /usr/lib/go/pkg/include /home/onoketa/tls-tris/_dev/GOROOT/linux_amd64/pkg/include
cp -HR /usr/lib/go/pkg/tool /home/onoketa/tls-tris/_dev/GOROOT/linux_amd64/pkg/tool
rm -r /home/onoketa/tls-tris/_dev/GOROOT/linux_amd64/src/crypto/tls/*
rsync -rltgoD /home/onoketa/tls-tris/ /home/onoketa/tls-tris/_dev/GOROOT/linux_amd64/src/crypto/tls/ --exclude=_dev
for p in /home/onoketa/tls-tris/_dev/patches/88253a956a753213617d95af3f42a23a78798473.patch; do patch -d "/home/onoketa/tls-tris/_dev/GOROOT/linux_amd64" -p1 < "$p"; done
patching file src/net/http/request.go
Hunk #1 succeeded at 294 (offset 19 lines).
patching file src/net/http/server.go
Hunk #1 succeeded at 241 with fuzz 1 (offset 18 lines).
Hunk #2 succeeded at 1009 (offset 34 lines).
Hunk #3 succeeded at 3329 (offset 159 lines).
perl -pi -e 's,"golang\.org/x/crypto/,"internal/x/crypto/,' /home/onoketa/tls-tris/_dev/GOROOT/linux_amd64/src/crypto/tls/*.go
perl -pi -e 's,"golang\.org/x/sys/cpu","internal/cpu",' /home/onoketa/tls-tris/_dev/GOROOT/linux_amd64/src/crypto/tls/*.go
perl -pi -e 's,"github\.com/,"github_com/,' /home/onoketa/tls-tris/_dev/GOROOT/linux_amd64/src/crypto/tls/*.go
git clone https://github.com/henrydcase/nobscrypto.git /tmp/tmp.lH15Fjoqf8/nobs
正克隆到 '/tmp/tmp.lH15Fjoqf8/nobs'...
remote: Enumerating objects: 2, done.
remote: Counting objects: 100% (2/2), done.
remote: Compressing objects: 100% (2/2), done.
remote: Total 901 (delta 1), reused 0 (delta 0), pack-reused 899
接收对象中: 100% (901/901), 1.09 MiB | 683.00 KiB/s, 完成.
处理 delta 中: 100% (494/494), 完成.
cd /tmp/tmp.lH15Fjoqf8/nobs; git checkout tags/Release_0.1
注意：正在检出 'tags/Release_0.1'。

您正处于分离头指针状态。您可以查看、做试验性的修改及提交，并且您可以通过另外
的检出分支操作丢弃在这个状态下所做的任何提交。

如果您想要通过创建分支来保留在此状态下所做的提交，您可以通过在检出命令添加
参数 -b 来实现（现在或稍后）。例如：

  git checkout -b <新分支名>

HEAD 目前位于 a456dc4 readme: License
perl -pi -e 's/sed -i/perl -pi -e/' /tmp/tmp.lH15Fjoqf8/nobs/Makefile
cd /tmp/tmp.lH15Fjoqf8/nobs; make vendor-sidh-for-tls
make[1]: 进入目录“/tmp/tmp.lH15Fjoqf8/nobs”
rm -rf /tmp/tmp.lH15Fjoqf8/nobs/build/
rm -rf tls_vendor
mkdir -p tls_vendor/github_com/henrydcase/nobs/
rsync -a . tls_vendor/github_com/henrydcase/nobs/ --exclude=tls_vendor --exclude=.git --exclude=.travis.yml --exclude=README.md
find tls_vendor -type f -print0 -name "*.go" | xargs -0 perl -pi -e 's/github\.com/github_com/g'
make[1]: 离开目录“/tmp/tmp.lH15Fjoqf8/nobs”
cp -rf /tmp/tmp.lH15Fjoqf8/nobs/tls_vendor/. /home/onoketa/tls-tris/_dev/GOROOT/linux_amd64/src/vendor/
git clone https://github.com/cloudflare/sidh.git /tmp/tmp.lH15Fjoqf8/sidh
正克隆到 '/tmp/tmp.lH15Fjoqf8/sidh'...
remote: Enumerating objects: 872, done.
remote: Total 872 (delta 0), reused 0 (delta 0), pack-reused 872
接收对象中: 100% (872/872), 585.21 KiB | 470.00 KiB/s, 完成.
处理 delta 中: 100% (560/560), 完成.
cd /tmp/tmp.lH15Fjoqf8/sidh; git checkout tags/Release_1.0
注意：正在检出 'tags/Release_1.0'。

您正处于分离头指针状态。您可以查看、做试验性的修改及提交，并且您可以通过另外
的检出分支操作丢弃在这个状态下所做的任何提交。

如果您想要通过创建分支来保留在此状态下所做的提交，您可以通过在检出命令添加
参数 -b 来实现（现在或稍后）。例如：

  git checkout -b <新分支名>

HEAD 目前位于 d5bd9b5 travis: compile also with go1.11 and on ARM
perl -pi -e 's/sed -i/perl -pi -e/' /tmp/tmp.lH15Fjoqf8/sidh/Makefile
cd /tmp/tmp.lH15Fjoqf8/sidh; make vendor
make[1]: 进入目录“/tmp/tmp.lH15Fjoqf8/sidh”
rm -rf /tmp/tmp.lH15Fjoqf8/sidh/build
rm -rf coverage*.txt
mkdir -p build/vendor/github_com/cloudflare/sidh/
rsync -a . build/vendor/github_com/cloudflare/sidh/ \
        --exclude=build/vendor \
        --exclude=.git          \
        --exclude=.travis.yml   \
        --exclude=README.md     \
        --exclude=Makefile      \
        --exclude=build
# This swaps all imports with github.com to github_com, so that standard library doesn't
# try to access external libraries.
find build/vendor -type f -iname "*.go" -print0  | xargs -0 perl -pi -e 's/github\.com/github_com/g'
make[1]: 离开目录“/tmp/tmp.lH15Fjoqf8/sidh”
cp -rf /tmp/tmp.lH15Fjoqf8/sidh/build/vendor/. /home/onoketa/tls-tris/_dev/GOROOT/linux_amd64/src/vendor/
GOARCH=amd64 GOROOT="/home/onoketa/tls-tris/_dev/GOROOT/linux_amd64" go install -v std
_dev/GOROOT/linux_amd64/src/crypto/tls/13.go:24:2: unknown import path "github_com/cloudflare/sidh/sidh": cannot find module providing package github_com/cloudflare/sidh/sidh
_dev/GOROOT/linux_amd64/src/crypto/tls/13.go:25:2: unknown import path "github_com/cloudflare/sidh/sike": cannot find module providing package github_com/cloudflare/sidh/sike
_dev/GOROOT/linux_amd64/src/vendor/github_com/cloudflare/sidh/p503/arith_decl.go:6:2: unknown import path "github_com/cloudflare/sidh/internal/isogeny": cannot find module providing package github_com/cloudflare/sidh/internal/isogeny
_dev/GOROOT/linux_amd64/src/vendor/github_com/cloudflare/sidh/p503/consts.go:5:2: unknown import path "github_com/cloudflare/sidh/internal/utils": cannot find module providing package github_com/cloudflare/sidh/internal/utils
_dev/GOROOT/linux_amd64/src/vendor/github_com/cloudflare/sidh/sidh/params.go:5:2: unknown import path "github_com/cloudflare/sidh/p503": cannot find module providing package github_com/cloudflare/sidh/p503
_dev/GOROOT/linux_amd64/src/vendor/github_com/cloudflare/sidh/sidh/params.go:6:2: unknown import path "github_com/cloudflare/sidh/p751": cannot find module providing package github_com/cloudflare/sidh/p751
_dev/GOROOT/linux_amd64/src/vendor/github_com/cloudflare/sidh/sike/sike.go:12:2: unknown import path "github_com/henrydcase/nobs/hash/sha3": cannot find module providing package github_com/henrydcase/nobs/hash/sha3
_dev/GOROOT/linux_amd64/src/vendor/github_com/henrydcase/nobs/dh/sidh/api.go:5:2: unknown import path "github_com/henrydcase/nobs/dh/sidh/internal/isogeny": cannot find module providing package github_com/henrydcase/nobs/dh/sidh/internal/isogeny
_dev/GOROOT/linux_amd64/src/vendor/github_com/henrydcase/nobs/dh/sidh/params.go:5:2: unknown import path "github_com/henrydcase/nobs/dh/sidh/p503": cannot find module providing package github_com/henrydcase/nobs/dh/sidh/p503
_dev/GOROOT/linux_amd64/src/vendor/github_com/henrydcase/nobs/dh/sidh/params.go:6:2: unknown import path "github_com/henrydcase/nobs/dh/sidh/p751": cannot find module providing package github_com/henrydcase/nobs/dh/sidh/p751
_dev/GOROOT/linux_amd64/src/vendor/github_com/henrydcase/nobs/kem/sike/sike.go:11:2: unknown import path "github_com/henrydcase/nobs/dh/sidh": cannot find module providing package github_com/henrydcase/nobs/dh/sidh
make: *** [_dev/Makefile:68：/home/onoketa/tls-tris/_dev/GOROOT/linux_amd64/.ok_go1.12.7_linux_amd64] 错误 1
```

**Comment by Lekensteyn** [comment]
Which go version are you using, and what is your current tris version?

Arch Linux currently ships with 1.12.7 which should work. 1.13beta1 might require some more changes, but I am working on that.

**Comment by Lekensteyn** [comment]
This should be fixed by #178 now. If not, please provide more details, thanks!
